go/executor/executor.go

```diff
@@ -225,6 +272,9 @@ func (s *pythonExecutor) runCommand(cmd *exec.Cmd, context map[string]string, lo
 }
 
 func (s *pythonExecutor) ExecuteQuery(stmt *ir.NormalStmt) error {
+	if ok, err := s.tryExperimentalExecute(stmt, false); ok {
```

Comment:
Can we have some FLAG to switch between modes?

Reply:
Added `SQLFLOW_USE_EXPERIMENTAL_CODEGEN` env.